docs/deeprank.tools.rst

```diff
@@ -5,7 +5,6 @@ Tools
 
 This module contains a series of core tools used for the feature calculations in DeepRank. These tools are at the moment:
 
-    - pdb2sql : a sql based parser of pdb
     - StructureSimilarity : compute irmsd,lrmsd,Fnat and DockQ score (Profit replacement)
```

Comment:
Structure similarity is also part of pdb2sql now so we should remove that part

Reply:
I actually don't quite know what will be left in that section then

Reply:
.swp and .swo files removed in commit [a504c3c2b7486835d98a681ad710c8372f77c43c](https://github.com/DeepRank/deeprank/commit/a504c3c2b7486835d98a681ad710c8372f77c43c) . 

Do you think we should remove both the docs/deeprank.tools.rst file and the deeprank/tools/README.md since it is [practically empty after I removed pdb2sql from it](https://github.com/DeepRank/deeprank/commit/60b23c2630c344425cc2938f272f8fad19d17c59)?

Reply:
As far as I can say the sasa in tools is only used in NaivePSSM that we shouldn't use. The sparse tool is used a bit so we have to keep it. So for now it's probably best to leave it as is and merge and  close this PR. However we should clean the code a bit and remove the unused parts (like sasa)

Reply:
As far as regards the sasa in particular, I would leave it because I need the sasa calculation (trough freesasa) for implementing the desolvation energy as a feature (Issue #68 ).